templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -100,8 +102,12 @@ contents:
             #!/bin/sh
             set -euxo pipefail
 
-            source /run/etcd/environment
+            if [ -e  "/var/lib/etcd/member/snap/db" ]; then
+              echo "etcd has previously been initialized as a member"
+              exit 0
```

Comment:
by exiting early we avoid unbound variable for ETCD_ENDPOINTS.

Reply:
I think in general we should move this init container logic to go, but for now this moves towards unblocking DR